src/core/eme/utils/loaded_sessions_store.ts

```diff
@@ -118,6 +118,21 @@ export default class LoadedSessionsStore {
                                    sessionType: entry.sessionType };
   }
 
+  /**
+   * Moves the corresponding MediaKeySession to the end of its internal storage,
+   * as returned by the `getAll` method.
+   *
+   * This can be used to signal that a previously-stored initialization data is
+   * re-used to then be able to implement a caching replacement algorithm based
+   * on the least-recently-used values by just evicting the first values
+   * returned by `getAll`.
+   */
+  public moveOnTop(
```

Comment:
This has the exact same effect than `getAndReuse` but I felt the name was not right when we don't care about the result.

I'm still not happy with the name though.

Reply:
Just renamed it to `reuse`